ksql-rest-model/src/main/java/io/confluent/ksql/rest/entity/HostStatusEntity.java

```diff
@@ -18,28 +18,26 @@
 import com.fasterxml.jackson.annotation.JsonCreator;
 import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
 import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.Map;
 import java.util.Objects;
 
 @JsonIgnoreProperties(ignoreUnknown = true)
 public class HostStatusEntity {
 
-  private HostInfoEntity hostInfoEntity;
   private boolean hostAlive;
   private long lastStatusUpdateMs;
+  private Map<String, ActiveStandbyEntity> perQueryActiveStandbyEntity;
 
   @JsonCreator
   public HostStatusEntity(
-      @JsonProperty("hostInfoEntity") final HostInfoEntity hostInfoEntity,
       @JsonProperty("hostAlive") final boolean hostAlive,
-      @JsonProperty("lastStatusUpdateMs") final long lastStatusUpdateMs
+      @JsonProperty("lastStatusUpdateMs") final long lastStatusUpdateMs,
+      @JsonProperty("perQueryActiveStandbyEntity")
+      final Map<String, ActiveStandbyEntity> perQueryActiveStandbyEntity
   ) {
-    this.hostInfoEntity = Objects.requireNonNull(hostInfoEntity, "hostInfoEntity");
     this.hostAlive = hostAlive;
     this.lastStatusUpdateMs = lastStatusUpdateMs;
-  }
-
-  public HostInfoEntity getHostInfoEntity() {
-    return hostInfoEntity;
+    this.perQueryActiveStandbyEntity = perQueryActiveStandbyEntity;
```

Comment:
nit: validate params that will be stored in object state; ensuring object does not get into an invalid state.

Reply:
Done